### ISSUE TITLE
examples/external-secrets does not sync external secrets

### ISSUE THREAD

**Issue by AureaMohammadAlavi** [aws-ia/terraform-aws-eks-blueprints-addons#185]

**Comment by AureaMohammadAlavi** [comment]
Sorry, I created the issue in the wrong repo. K8s service account for the external secret controller is created by the helm chart. However `ClusterSecretStore` and `SecretStore` resources are using two separate service accounts (`cluster-secretstore-sa` and `secretstore-sa`), which were used to be created by the local irsa module, but after migrating to the `iam-role-for-service-accounts-eks` module these two service accounts are NOT being created.

**Comment by bryantbiggs** [comment]
yes, I moved the issue here since this is where the implementation lives for the addon. We will look at fixing this here and making the appropriate changes in the respective examples
